README.md

```diff
@@ -4,15 +4,22 @@ Steam from Raspberry Pi to Django website with WebSockets
 
 ### Setup
 1. Enable camera interface `sudo raspi-config`
-2. Enter user folder `cd /home/pi`
+2. Enter user folder if you are not already there `cd /home/pi`
 3. Clone repo `git clone https://github.com/MAKENTNU/stream.git`
 4. Enter stream folder `cd stream`
-5. Add local settings `nano local_settings.py`
 
 The rest of the steps can be completed by running the setup script `./setup.sh`.
 It will ask for the stream name, which can only consist of english letters, numbers, hyphens or underscores.
 The secret can be found on locally on the MAKE NTNU server in `local_settings.py` and is called `stream_key`.
 
+The pi will reboot once the script is done and the stream should now be visible on the website.
+
+The stream name and secret can be edited manually if needed:
```

Comment:
This sentence should probably be changed to something like the following, to minimize any potential confusion 🙂 
```suggestion
The `STREAM_NAME` and `KEY` can be edited manually if needed:
```